app/models/loader/types.rb

```diff
@@ -96,43 +96,58 @@ def create_resource!
           resource.annotations_dataset.import(%i(resource_id name value), records)
         end
       end
-      
+
```

Comment:
I'll add a plus one to @micahlee 's comment from the other PR that all formatting changes should be moved into their own commit.  You had mentioned:

> Its not mine - came from merge

If you do a rebase, instead of a merge (which you always should to bring your branch up to date with master) then this won't happen.  There are ways you can still fix this in git that won't require manual intervention on your part.  Let me know if you need any help with that.

Btw, is there a difference between this and #1790 ?

Reply:
There is not difference between these commits - only technical problems with merge. If we will use the old one - we will loose Rafi's comments..